DataDog/documentation
# Fetch IA subdomain from SSM

### What does this PR do? What is the motivation?
https://datadoghq.atlassian.net/browse/WEB-4232

This allows the IA subdomain to be fetched from SSM at build time, rather than hardcoded. This will allow us to share the subdomain value across multiple sites, making it easier to update when ad blockers block the current RUM intake url.

Current SSM value is `iam-rum-intake`

### Merge instructions

**How to test:**
1. Visit https://docs-staging.datadoghq.com/ccole/ia-subdomain/
2. In the dev console, search network requests for `rum` and make sure the request URL is https://iam-rum-intake.datadoghq.com


### Additional notes
